### DISCUSSION TITLE
Ability to change font size

### DISCUSSION THREAD

**Discussion by jamesbhatta** [laradumps/app#329] [Ideas]
# 🛻 LaraDumps Feature Request

## Summary

I don't see any setting for changing font size. It would be great if we can.

### Why is this needed?

I am getting pretty small font in Laradump's window than other apps but not being able to increase its font size.

### Suggested Solution(s)

A font size setting in settings page.

### Related issue(s)/PR(s)

N/A

**Comment by mardr** [comment]
For me, the default font size was also too small. Use Ctrl + touchpad scroll up/down or Ctrl + mouse wheel to change the text size.
